### ISSUE TITLE
MasterNodeConfig is missing Node property

### ISSUE THREAD

**Issue by timbussmann** [Particular/NServiceBus#4120]
In v5, `MasterNodeConfig` contains a `Node` property: https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/Distributor/MasterNode/MasterNodeConfig.cs

This section has been deprecated in v6 and we have code which checks the usage of the section at startup. But the `Node` property has been removed (because the class was obsoleted) which causes an exception when starting an endpoint after migration to v6 if it contains that section in the app.config with a node setting (always...). This exception is thrown because of the "invalid attribute" not supported by the config section and it's shown before we are able to throw our exception about the obsoleted section (which is much more user friendly).

TL;DR;
we should re-add the Node property for our custom exception to properly help users to remove the section when upgrading.

**Comment by danielmarbach** [comment]
@timbussmann can we close this since #4124 is merged?

**Comment by timbussmann** [comment]
yep, good for closing. Verified locally that the exceptions are now the more helpful obsolete messages we throw from NServiceBus :+1:
